pkg/osgi_config_manager.go

```diff
@@ -20,6 +21,22 @@ func (cm *OSGiConfigManager) ByPID(pid string) OSGiConfig {
 	return OSGiConfig{manager: cm, pid: pid}
 }
 
+func (cm *OSGiConfigManager) ByFactoryPID(pid string) OSGiConfig {
+	factoryPid, suffix := getFactoryPID(pid)
+	return OSGiConfig{manager: cm, pid: pid, fpid: factoryPid, aemcId: suffix}
+}
+
+func getFactoryPID(pid string) (string, string) {
```

Comment:
`get` prefix is reserved for getters only; if there is some computation/parsing in methods I prefer to use more specialized naming like `extract` `determine` `parse` `split` etc

Reply:
how about just `splitPID()` ?